content/blog/asyncapiathacktoberfest.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "AsyncAPI @Hacktoberfest"
+date: 2020-09-30T06:00:00+01:00
+type: blog
+featured: true
+tags:
+  - Hacktoberfest
+  - Community
+  - Contribute
+cover: /images/posts/
+weight: 10
+authors:
+  - name: Lukasz Gornicki
+    photo: /images/avatars/lpgornicki.webp
+    link: https://twitter.com/derberq
+    byline: AsyncAPI Maintainer and Community Guardian
+---
+
+![cover](/images/posts/)
+
+## What is AsyncAPI
+
+AsyncAPI is a specification for describing your [event-driven architecture](https://www.asyncapi.com/docs/getting-started/event-driven-architectures/). You are probably using already [OpenAPI/Swagger specification](https://www.asyncapi.com/docs/getting-started/coming-from-openapi/) for describing your synchronous RESTful APIs. AsyncAPI is something that supplements OpenAPI. When you should use AsyncAPI, one of the cases is when your services do not talk to each other directly but through a message broker.
```

Comment:
> When you should use AsyncAPI, one of the cases is when your services do not talk to each other directly but through a message broker.

What? 😄

Reply:
`through a message broker` like `over a message broker` or 🤔 
What is wrong with this sentence?

Reply:
> When you should use AsyncAPI, one of the cases is when your services

The beginning is confusing. Looks like if you were trying to join two sentences and ended up like this by mistake. I'll explain more. It says "When you should use AsyncAPI ..... is when your services". I'm not sure it's incorrect or not but looks super weird. Maybe you can rephrase it as folllows:

```suggestion
AsyncAPI is a specification for describing your [event-driven architecture](https://www.asyncapi.com/docs/getting-started/event-driven-architectures/). You are probably using already [OpenAPI/Swagger specification](https://www.asyncapi.com/docs/getting-started/coming-from-openapi/) for describing your synchronous RESTful APIs. AsyncAPI is something that supplements OpenAPI. As an example, you should use AsyncAPI when your services do not talk to each other directly but through a message broker.
```

Reply:
ah, ok, I like your suggestion